lib/turn.rb

```diff
@@ -0,0 +1,21 @@
+class Turn
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    @guess == @card.answer
+  end
+
+  def feedback
+    if correct?
```

Comment:
Great job using the previously defined `correct?` method to your advantage and not re-writing the same line over again.